src/main/java/digital/slovensko/autogram/ui/gui/GUIApp.java

```diff
@@ -31,6 +32,8 @@ public void start(Stage windowStage) throws Exception {
 
         windowStage.setOnCloseRequest(event -> {
             new Thread(server::stop).start();
+            timer.cancel();
```

Comment:
Toto mi pride ako prilisna vnutornost. Nemal by mat mat autogram len nejaky close kde sa toto udeje?

Reply:
tomuto nerozumiem. Vytvára a zastavuje sa to na rovnakom mieste ako server. To nie je ok?